addons/account/views/account_move_views.xml

```diff
@@ -645,6 +645,16 @@
                                 attrs="{'invisible' : [('to_check', '=', False)]}" data-hotkey="k" />
                         <field name="state" widget="statusbar" statusbar_visible="draft,posted"/>
                     </header>
+                    <div class="alert alert-warning mb-0" role="alert"
+                         attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('duplicated_ref', '=', [])]}">
+                        Warning: Duplicated bill:
+                        <button name="open_duplicated_ref_bill_view"
+                                type="object"
+                                string="See duplicated bill(s)"
+                                icon="fa-arrow-right"
+                                class="none"
```

Comment:
class is set to avoid the button being styled.

Reply:
You should put the correct class instead (`btn-primary`, `btn-secondary`, `btn-link`)
`none` might be something else, and it also hints to `display: none`

Reply:
> You should put the correct class instead (`btn-primary`, `btn-secondary`, `btn-link`) `none` might be something else, and it also hints to `display: none`

Can I just write `class=""`?

Reply:
Note sure what you are trying to do here :shrug:

Reply:
Thx for your help !